src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -5602,6 +5602,49 @@ public override IOperation VisitIsPattern(IIsPatternOperation operation, int? ca
                 operation.Syntax, operation.Type, operation.ConstantValue, IsImplicit(operation));
         }
 
+        public override IOperation VisitInvalid(IInvalidOperation operation, int? captureIdForResult)
+        {
+            if (_currentStatement == operation)
+            {
+                VisitInvalidOperationStatement(operation);
+                return null;
+            }
+            return VisitInvalidOperationExpression(operation);
+        }
+
+        private void VisitInvalidOperationStatement(IInvalidOperation operation)
+        {
+            Debug.Assert(_currentStatement == operation);
+            VisitStatements(operation.Children);
```

Comment:
>VisitStatements(operation.Children); [](start = 12, length = 36)

This means that, if there are no children, the entire node will disappear. This feels wrong. The VisitNoneOperationStatement has the same problem, BTW. We probably should add another statement with an invalid operation that is similar to the ```operation```, but with all children removed. And probably do the same for VisitNoneOperationStatement, only create operation None there.